modules/microshift-adding-app-rpms-to-blueprint.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-adding-app-rpms-to-blueprint_{context}"]
+= Adding application RPMs to a blueprint
+
+.Prerequisites
+* You have root access to the host.
+* Workload or application RPMs exist in the `~/rpmbuild/RPMS` directory.
+
+To add application RPMs to a blueprint, you must create a local repository that Image Builder can use to create the ISO. With this procedure, the required container images for your workload can be pulled over the network.
+
+. Create a local RPM repository by running the following command:
++
+[source,terminal]
+----
+$ createrepo ~/rpmbuild/RPMS/
+----
+
+. Give Image Builder access to the RPM repository by running the following command:
++
+[source,terminal]
+----
+$ sudo chmod a+rx ~
+----
++
+[NOTE]
+====
+You must ensure that Image Builder has all of the necessary permissions to access all of the files needed for image building, or the build cannot proceed.
+====
++
+. Create the blueprint file, `repo-local-rpmbuild.toml`, with this example content:
++
+[source,terminal]
+----
+id = "local-rpm-build"
+name = "RPMs build locally"
+type = "yum-baseurl"
+url = "file://<PATH>/rpmbuild/RPMS" <1>
+check_gpg = false
+check_ssl = false
+system = false
+----
+<1> Specify part of the path to create a location that you choose. Use this path in the later commands to set up the repository and copy the RPMs.
+
+. Add the repository as a source for Image Builder by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli sources add repo-local-rpmbuild.toml
+----
+
+. Add the RPM to your `.toml` blueprint file, by adding the following lines:
++
+[source,terminal]
+----
+…
+[[packages]]
+name = "<application-workload-manifests>" <1>
+version = "*"
+…
+----
+[.small]
+<1> Add the name of your workload here.
+
+. Push the updated blueprint to Image Builder by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli blueprints push repo-local-rpmbuild.toml
+----
+
+. At this point, you can either run Image Builder to create the ISO, or embed the container images for offline use.
```

Comment:
Here, in the preview seen as step 7, the reader is told they have two choices. However, only subsection a. is present below, for "creating the ISO". It seems incomplete that there is no subsection b. to address "embed the container images"?

Reply:
This is a separate procedure, not a step b. The setup here is docs style and correct.